msdk-featdet/msdk-featdet-openms/src/main/java/io/github/msdk/featdet/openms/FeatureFinderMetaboLocator.java

```diff
@@ -0,0 +1,90 @@
+/* 
+ * (C) Copyright 2015-2016 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+
+package io.github.msdk.featdet.openms;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import io.github.msdk.MSDKException;
+
+public class FeatureFinderMetaboLocator {
+
+	private final static String OPENMS_FEATURE_FINDER_METABO_LIBRARY_NAME = "FeatureFinderMetabo";
+	private final static String OPENMS_DEFAULT_LOCATION_WINDOWS = "C:\\Program Files\\";
+	private final static String OPENMS_DEFAULT_LOCATION_MAC = "/Applications/";
+	private final static String OPENMS_DEFAULT_LOCATION_LINUX = "/usr/local/";
+	private final static String FEATURE_FINDER_METABO_EVAL = "No options given. Aborting!";
+
+	public static String findFeatureFinderMetabo() throws MSDKException {
+		if (!isFeatureFinderMetaboHere(OPENMS_FEATURE_FINDER_METABO_LIBRARY_NAME)) {
+			return OPENMS_FEATURE_FINDER_METABO_LIBRARY_NAME;
+		} else {
+			String OS = System.getProperty("os.name").toLowerCase();
```

Comment:
Although not necessary, for variables that are not going to change, it is a good practice to use the _final_ keyword (final String OS = ...)
http://stackoverflow.com/questions/137868/using-final-modifier-whenever-applicable-in-java